tests/test.bats

```diff
@@ -22,7 +22,7 @@ teardown() {
   cd ${TESTDIR}/${PROJNAME}
   echo "# ddev get ${DIR} with project ${PROJNAME} in ${TESTDIR}/${PROJNAME} ($(pwd))" >&3
   ddev get ${DIR}
-  ddev config --nodejs-version 18 --project-type=drupal10
+  ddev config --nodejs-version 18
```

Comment:
```suggestion
  ddev config --auto
```

IMO it would be better to leave DDEV's default node.js (currently 20, which is LTS) unless there's a strong reason to do otherwise.